pkg/grpc/vehicle_template.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v4.25.0
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.24.3
```

Comment:
update your version of protoc, i think it should be brew upgrade protobuf

Reply:
thanks!